src/lib/TimeSeries/timeSeries.ts

```diff
@@ -36,6 +36,9 @@ export class Series {
   start?: Date;
   end?: Date;
   lastUpdated?: Date;
+  domains?: any[]; 

```

Comment:
Can we add an explicit type to this?

Reply:
Seems to be `[string][]` (i.e. array of 1-tuples with strings)

Reply:
is `Domain[]`, added